index.html

```diff
@@ -1,286 +1,430 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>  
-  <link rel="stylesheet" href="./css/style.css">
-  <link rel="stylesheet" href="./css/style-desktop.css">
-  <title>Portfolio</title>
-</head>
-<body>
-  <header class="header--section">
-    <a href="#" class="logo">Jos</a>
-    <nav class="nav--container">
-      <ul class="nav-menu d-flex">
-        <li class="nav-link"><a href="#works">Portoflio</a></li>
-        <li class="nav-link"><a href="#about">About</a></li>
-        <li class="nav-link"><a href="#contact">Contact</a></li>
-      </ul>
-    </nav>
-    <span  class= "nav-humburger" id="humburger"><i class="fa humburger fa-bars"></i></span>
-  </header>
```

Comment:
- [ ] The mobile version of your app is well-built, but when you are in the desktop version, the hamburger menu button should be ```invisible```.
I will suggest you to use the following syntax in the desktop version :ear: 
```bash
.nav-humburger {
        display: none;
}
```